SlowCheetah/Transformer/XmlTransformer.cs

```diff
@@ -13,6 +13,25 @@ namespace SlowCheetah
     /// </summary>
     public class XmlTransformer : ITransformer
     {
+        private IXmlTransformationLogger logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="XmlTransformer"/> class.
+        /// </summary>
+        public XmlTransformer()
+            : this(null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="XmlTransformer"/> class with an external logger.
+        /// </summary>
+        /// <param name="logger">External logger. Passed into the transformation</param>
+        public XmlTransformer(IXmlTransformationLogger logger)
```

Comment:
With the new `ITransformationLogger` interface, you should get rid of this constructor and shim to `IXmlTransformationLogger`